ui/package.json

```diff
@@ -16,36 +16,36 @@
   "dependencies": {
     "@heroicons/react": "^2.0.11",
     "@metamask/eth-sig-util": "^5.0.0",
-    "@types/node": "^17.0.34",
-    "@types/react-dom": "^18.0.4",
-    "daisyui": "^2.13.5",
+    "@types/node": "^18.7.23",
+    "@types/react-dom": "^18.0.6",
+    "daisyui": "^2.31.0",
```

Comment:
I think this upgrade caused the menu layout to change?

Reply:
I'll look into this. And make changes accordingly.